freqtrade/resolvers/iresolver.py

```diff
@@ -0,0 +1,68 @@
+# pragma pylint: disable=attribute-defined-outside-init
+
+"""
+This module load custom objects
+"""
+import importlib.util
+import inspect
+import logging
+from pathlib import Path
+from typing import Optional, Dict, Type, Any
+
+logger = logging.getLogger(__name__)
+
+
+class IResolver(object):
+    """
+    This class contains all the logic to load custom hyperopt class
+    """
+
+    def __init__(self, object_type, config: Optional[Dict] = None) -> None:
```

Comment:
what good does it do object_type here ?

Reply:
good point, the whole __init__ is not needed, as we need to subclass this anyway